packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/isRightFlankingFor..st

```diff
@@ -0,0 +1,21 @@
+testing-private
+isRightFlankingFor: aMarkdownReadStream
+
+	"1 not preceded by whitespace AND"
+	"   2a not preceded by punct OR"
+	"	2b preceded  by punct and followed by ws or punct"
```

Comment:
should we keep this comment?

Reply:
It says what is written in code, so remove it. You could add a reference to the spec, but I don't think it would help. Also, you could make use of `==>` in the return expression (but if you think that matching wordings matter more, that's fine too)